aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -2493,14 +2493,17 @@ TORCH_IMPL_FUNC(linalg_vector_norm_out)(const Tensor& self, const Scalar& scalar
   norm_stub(iter.device_type(), iter, ord);
 }
 
-void _linalg_matrix_norm_checks(const Tensor& A, IntArrayRef dim, optional<ScalarType> opt_dtype) {
-  at::native::checkFloatingOrComplex(A, "linalg.matrix_norm");
-  TORCH_CHECK(A.dim() >= 2,
-              "linalg.matrix_norm: input tensor must be a matrix or a batch of matrices");
+void _linalg_matrix_norm_checks(const Tensor& A, std::vector<int64_t>& dim, optional<ScalarType> opt_dtype, bool low_precision) {
+  // A
```

Comment:
stray comment?

Reply:
In this function I perform the checks for `A`, `dim` and  `dtype`. This is just a comment indicating what are the preconditions in each input, but yeha, I agree that I could've been a bit more verbose :D